### PR TITLE
Reuse copy buttons path

### PR DESCRIPTION
**Pull request by mondeja**
Reduces HTML file size from 2.37 MB to 1.66 MB.

### CODE REVIEW

**Comment by sachinraja** [comment]
Not sure which browsers we're targeting but [`xlink:href` is deprecated](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use#attributes)

**Comment by mondeja** [comment]
Great, it seems to work with `href`, so 6 characters less for each icon.

**Comment by sachinraja** [comment]
I'm not very familiar with it, but I think we should put the symbol between [`defs`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs) instead of using `display: none` on the svg.

**Comment by mondeja** [comment]
> I'm not very familiar with it, but I think we should put the symbol between [`defs`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs) instead of using `display: none` on the svg.

I've tested it and doesn't works, it adds a huge empty space at the top of the page. Seems that the dimensions must be defined.

This works:

```pug
svg(xmlns='http://www.w3.org/2000/svg', width="0", height="0")
  defs
    symbol#cp
      path(d='...')
```

And this:

```pug
svg(xmlns='http://www.w3.org/2000/svg', width="0", height="0")
  symbol#cp
    path(d='...')
```

But not this:

```pug
svg(xmlns='http://www.w3.org/2000/svg')
  defs
    symbol#cp
      path(d='...')
```

I'm fine with either way. Chosen this approach because `width="0", height="0"` is two characters longer than `style="display:none"` and seems more descriptive of what means. Added `defs` in https://github.com/simple-icons/simple-icons-website/pull/174/commits/ed99f3a494c271f74101d46298072ba7dbeaaa9b because [SVG spec v2 recommends it](https://svgwg.org/svg2-draft/struct.html#DefsElement) for accesibility reasons.

**Review by sachinraja — APPROVED** [review]

File: public/index.pug [hunk 1]
```diff
@@ -30,6 +30,12 @@ html(lang='en-US')
     )
     link(rel='license', href='license.txt')
     link(rel='canonical', href=pageUrl)
+    svg(xmlns='http://www.w3.org/2000/svg')
```
Comment (sachinraja) [inline public/index.pug:33]: Sorry, may have missed something in your comment. Should this have the `width="0"` and `height="0"` attributes?

Comment (mondeja) [reply public/index.pug:33]: The SVG is hidden with a combination of `width="0"` and `height="0"` or just using `style="display:none"`, so it is the same. The SVG spec doesn't recommends any choice.

Comment (mondeja) [reply public/index.pug:33]: Ups, seems that I've forgotten the `style="display:none"`. Fixed by #177

**Pull request merged**